app/views/videos/index.html.haml

```diff
@@ -1,9 +1,9 @@
 - @categories.each do |category|
-  %article.video_category
+  %article.video_category(id="#{category.name.downcase}")
```

Comment:
This kind of assumes category name is unique. Otherwise whatever JS code hooks on this will not work properly. Also the name must not be composed of multiple words.